internal/website/content/howto/blob/data.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Store and Access Unstructured Data"
+date: 2019-03-21T08:42:51-07:00
+draft: true
+weight: 2
+---
+
+Once you have [opened a bucket][] for the storage provider you want, you can
+store and access data from them using the standard Go I/O patterns.
```

Comment:
"store and access data in it"?

[singular, and also "it" fits both store and access]

Reply:
Done.